controlplane/controllers/machine_controller.go

```diff
@@ -130,7 +130,7 @@ func (r *MachineReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 			}
 			if !etcdRemoved {
 				logger.Info("wait k3s embedded etcd controller to remove etcd")
-				return ctrl.Result{Requeue: true}, err
+				return ctrl.Result{RequeueAfter: etcdRemovalRequeueAfter}, err
```

Comment:
can we change err to nil? 

```return ctrl.Result{RequeueAfter: etcdRemovalRequeueAfter}, nil```

Reply:
err is already `nil`, but we could also set it explicitly, to be super clear that there is no err